huaweicloud/config/endpoints.go

```diff
@@ -604,6 +605,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Version: "v1",
 		Product: "DataArtsStudio",
 	},
+	"dataartsv1": {
```

Comment:
please make the key more meaningful.

```suggestion
	"dataarts-dlf": {
```